fuel-tests/tests/example_tx.json

```diff
@@ -1,5 +1,6 @@
 {
     "Script": {
+        "discriminant": "Script",
```

Comment:
should this be removed?

Reply:
Yep, the `fuel-tx` was not updated to the latest version=)